src/net/p2p.rs

```diff
@@ -643,7 +643,7 @@ impl PeerNetwork {
         let num_outbound = PeerNetwork::count_outbound_conversations(&self.peers);
         if !outbound && (self.peers.len() as u64) - num_outbound >= self.connection_opts.num_clients {
             // too many inbounds 
-            info!("{:?}: Too many inbound connections", &self.local_peer);
+            debug!("{:?}: Too many inbound connections", &self.local_peer);
```

Comment:
This should be reverted to `info!`.  Too many inbound connections is a sign of a DDoS attack, and should be shown in the logs by default.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1319/commits/868191d59d514ce9aac6ff55da764a9cfefe8aca